client/driver/raw_exec_test.go

```diff
@@ -265,7 +265,7 @@ func TestRawExecDriver_HandlerExec(t *testing.T) {
 		Driver: "raw_exec",
 		Config: map[string]interface{}{
 			"command": testtask.Path(),
-			"args":    []string{"sleep", "9000"},
+			"args":    []string{"sleep", "9000s"},
```

Comment:
This looks like it was fun to debug. :)

Reply:
Took me so long! Literally went home after I found the bug. I was just done haha